lib/vendor/bibtex_ruby.rb

```diff
@@ -37,8 +37,8 @@ def self.namecase_bibtex_entry(bibtex_entry)
         bibtex_entry.names.each do |n|
           n.first = NameCase(n.first) if n.first
           n.last = NameCase(n.last) if n.last
-          n.prefix = NameCase(n.prefix) if n.prefix
-          n.suffix = NameCase(n.suffix) if n.suffix
+          #n.prefix = NameCase(n.prefix) if n.prefix
```

Comment:
Why these changes?

Reply:
we had problems with capitalized author prefixes and suffixes. I do not think those should be there. Last and first names should be capitalized. Suffix and preffix should stay as it is.

Reply:
Namecase is smart, some prefix/suffix need capitalization.  Can we add a spec to illustrate problem?

Reply:
I would rely on what people enter to the DB. I run into similar problem with `Serial` capitalization. There is function for smart capitalization which keeps "a", "the", "with", "on" and other parts lower case. The problem only is, it knows only English grammer. For other languages, the title get completely distorted. I had to remove this capitalization. Prefix/suffix capitalization is also dependant on language, and, in some cases, on author preferences. Yes, you can make generic rules, but I would prefer to keep it simple.

Reply:
We can turn this on/off in the interface if required already.  Problematic records are a minority.

We have this because we import from crossref and other bibtex datasets that may have all caps etc., i.e. it fixes big batches of problems.   I don't want to have to manually downcase many many records on batch load.

Reply:
In this case the best if we better restrict the functionality to cases when the Author and Editors are not parsed as people and Journals not parsed as Serial. I have problem that you take time to  curate people, but than it get replaced by some other logic.